examples/injection/proto/injection.proto

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+import "oak/proto/handle.proto";
+
+package oak.examples.injection;
+
+message BlobStoreInterface {
+  oak.handle.Sender sender = 1 [(oak.handle.message_type) = ".oak.examples.injection.BlobRequest"];
+  oak.handle.Receiver receiver = 2
+      [(oak.handle.message_type) = ".oak.examples.injection.BlobResponse"];
+}
+
+message GetBlobRequest {
+  fixed64 id = 1;
+}
+
+message PutBlobRequest {
+  // Put 0 for new blob to store.
+  fixed64 id = 1;
+  string blob = 2;
+}
+
+message BlobRequest {
+  oneof request {
+    GetBlobRequest get = 1;
+    PutBlobRequest put = 2;
+  }
+}
```

Comment:
IIUC, this must be kept in sync with the service definition below, right?

Edit: it seems that this file has the service definition, which is the source of truth, and a bunch of helper messages that are currently manually generated, but ideally they could be generated automatically; I think I described something very similar here: https://github.com/project-oak/oak/issues/1192#issuecomment-648888063 . Even if out of scope for now, I think it would be good to clearly separate the source of truth vs what is currently manually implemented but in the future may be automatically generated. (Just a comment in the middle of this file, and moving things before / after the comment is enough for now of course)

Reply:
That is correct yes.

This is a very good point, I have added some comments that should cover this.